src/ducks/apps/index.js

```diff
@@ -465,45 +467,69 @@ function initializeRealtime() {
   }
 }
 
+async function _getInstalledInfos(app) {
+  try {
+    let installedApp = await cozy.client.fetchJSON(
```

Comment:
This information should already be in the redux store ? I don't see any reason to perform a fetch just to know if an app is installed. But maybe I am missing a point.

Reply:
The intent don't fetch all apps as the main application does. So we have to fetch this information here